frontier-chain/pallets/living-assets-evolution/src/lib.rs

```diff
@@ -223,10 +118,66 @@ impl<T: Config> Pallet<T> {
 		// we also use the last 12 bytes of the slot, since the first 4 bytes are always 0
 		bytes[..12].copy_from_slice(&slot_bytes[4..]);
 
-		let account_id_bytes = owner.as_fixed_bytes();
+		let mut owner_bytes = owner.encode();
+		owner_bytes.reverse();
+
+		// we assume that the `AccountId` is 20 bytes long. not relevant for tests
+		#[cfg(not(test))]
```

Comment:
Our goal is to test the production code as it is, to ensure it behaves as expected in real-world scenarios. Inserting code that only executes during testing could potentially mask issues that may occur in production.

Reply:
yep. The code that you are testing and the one in production differs.